### PR TITLE
add peer exchange nwaku config

### PR DESCRIPTION
**Pull request by LordGhostX**
- [x] add a guide on configuring peer exchange in nwaku
- [x] fix command line options assignment inconsistency (`:` to `=`)

### CODE REVIEW

File: docs/overview/concepts/peer-exchange.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Peer Exchange
 ---
 
-The primary objective of this protocol is to facilitate peer connectivity for resource-limited devices. The peer exchange protocol enables lightweight nodes to request peers from other nodes within the network. Light nodes can bootstrap and expand their mesh independently without relying on `Discv5`. Please refer to the [Peer Exchange](https://rfc.vac.dev/spec/34/) specification to learn more.
+The primary objective of this protocol is to facilitate peer connectivity for resource-limited devices. The peer exchange protocol enables light nodes to request peers from other nodes within the network. Light nodes can bootstrap and expand their mesh independently without relying on `Discv5`. Please refer to the [Peer Exchange](https://rfc.vac.dev/spec/34/) specification to learn more.
```
Comment (jm-clius) [inline docs/overview/concepts/peer-exchange.md:5]: Perhaps important to highlight here that this allows requesting _random_ peers from other nodes within the network, without requiring those nodes to divulge anything about their own connectivity or neighborhood. This is important from security perspective.

**Review by jm-clius — APPROVED** [review]
LGTM! Thanks. Just one comment on clarifying security property of peer exchange

**Review by Ivansete-status — COMMENTED** [review]
Thanks for this! It looks superb!
I just added a question that I'd like to clarify :)

File: docs/guides/nwaku/configure-discovery.md [hunk 2]
```diff
@@ -49,34 +49,64 @@ For instance, consider a `nwaku` node that enables `DNS Discovery`, connects to
...
+
+```bash title="Client: Nwaku Node Bootstrapping with Peer Exchange"
+./build/wakunode2 \
+  --tcp-port=30305 \
+  --ports-shift=1 \
+  --peer-exchange-node=/ip4/0.0.0.0/tcp/60000/p2p/16Uiu2HAmLCe6zVqCS6KMqqRbbhyoJjfYZGr1Q3thTSbyKzibQkFR
+```
+
+:::info
+`nwaku` provides a [`relay-peer-exchange`](/guides/reference/node-config-options#relay-config) option via `libp2p` for peer exchange, allowing network growth through neighboring nodes. However, this feature can compromise security and network robustness, so we recommend only using it in high-trust environments.
```
[the remark this answers is not in the document]
Comment (LordGhostX) [reply docs/guides/nwaku/configure-discovery.md:111]: Hanno: https://discord.com/channels/1110799176264056863/1126439361492287508/1126458375081824287

File: docs/guides/nwaku/configure-discovery.md [hunk 3]
```diff
@@ -49,34 +49,64 @@ For instance, consider a `nwaku` node that enables `DNS Discovery`, connects to
...
 :::info
 When Discv5 is enabled and used with [DNS Discovery](#configure-dns-discovery), the `nwaku` node will attempt to bootstrap the Discv5 routing table by extracting `ENRs` from peers discovered through DNS.
+:::
+
+## Configure Peer Exchange
+
+To enable [Peer Exchange](/overview/concepts/peer-exchange) in a `nwaku` node, use the following configuration options:
+
+- `peer-exchange`: Enables `Peer Exchange` on the node as a responder (disabled by default).
+- `peer-exchange-node` (optional): ENR for bootstrap node that has the peer exchange protocol enabled.
```
Comment (fryorcraken) [inline docs/guides/nwaku/configure-discovery.md:89]: ENR or Multiaddr? I am guessing multiaddr.

**Pull request merged**